rpmio/rpmlua.c

```diff
@@ -883,6 +883,30 @@ static int rpm_unsplitargs(lua_State *L)
     return 1;
 }
 
+static int rpm_glob(lua_State *L)
+{
+    const char *pat = luaL_checkstring(L, 1);
+    rpmglobFlags flags = RPMGLOB_NONE;
+    int argc = 0;
+    ARGV_t argv = NULL;
+
+    if (luaL_optstring(L, 2, "c"))
+	flags |= RPMGLOB_NOCHECK;
```

Comment:
Yup, see my earlier comment about that, "c" is kinda misleading but there could be more than one such disabler so we'd run out of "n"'s real fast :smile:

Reply:
Oh, I didn't mean to use "n" of course, but to make "c" mean "don't enable RPMGLOB_NOCHECK" :smile: But since that should be what most callers want by default, most callers would have to include "c" in there... which, otoh, may not be that terrible?

Reply:
Anyway, as we settled this via chat (to keep the patch as is), consider the above resolved :)